docs/add_content_and_edit_views.md

```diff
@@ -0,0 +1,266 @@
+# Add content and edit views
+
+To able to add and edit Content Item with our Field Type using PlatformUI, we need to create an edit view.
+The content view will also be needed to display correct information when viewing Content Item in "Content structure" tab in PlatformUI.
+
+## Implement edit view
+ 
+### Template file
+
+First, we need to add template file that will be responsible for displaying an edit form:
```

Comment:
First, we need to add **a** template file that will be responsible for displaying an edit form: